src/diode.js

```diff
@@ -18,15 +25,26 @@ function Diode(target) {
    * Callbacks are eventually executed, Diode does not promise
    * immediate consistency so that state propagation can be batched
    */
-  var _flush = function() {
+  var _flush = function(args) {
     /**
      * Important: do not cache the length of _callbacks
      * in the event a callback causes later subscriptions
      * to disappear
      */
     for (var i = 0; i < _callbacks.length; i++) {
-      _callbacks[i].apply(target, arguments)
+      _callbacks[i].apply(target, args)
+    }
+  }
+
+  var _cancel = function() {
+    var now = +new Date() // IE8 love
+
+    if (_lastFire && now - _lastFire < 10) {
+      cancelAnimationFrame(_tick)
```

Comment:
I'm trying to bundle together change events into a single request that only fires once per frame. `cancelAnimationFrame` was "bundling" by killing previous events, but it wasn't firing every frame because it kept canceling them. The result is that calling `volley()` constantly will never emit an event because they keep getting cancelled.